i18n/en.json

```diff
@@ -4,10 +4,12 @@
 	},
 	"ext-whowrotethat-activation-link": "Who Wrote That?",
 	"ext-whowrotethat-activation-link-tooltip": "Activate Who Wrote That?",
-	"ext-whowrotethat-state-pending": "<strong><em>Who Wrote That?</em></strong> is loading. This might take a while",
-	"ext-whowrotethat-state-error": "Error: $1",
-	"ext-whowrotethat-state-error-generic": "Could not load WhoWroteThat. Please refresh and try again.",
+	"ext-whowrotethat-state-pending": "<strong><em>Who Wrote That?</em></strong> is loading. This might take a while.",
+	"ext-whowrotethat-state-error": "API Error: $1",
+	"ext-whowrotethat-state-error-refresh": "Please refresh the page or try again later.",
+	"ext-whowrotethat-state-error-later": "Please try again later.",
+	"ext-whowrotethat-state-error-contact": "Please $1 about this issue.",
+	"ext-whowrotethat-state-error-contact-link": "contact us",
```

Comment:
Super pedantic comment that you should feel free to ignore :stuck_out_tongue: --

I'd rename those messages to be something like `ext-whowrotethat-error-refresh` and `ext-whowrotethat-error-contact` etc because `ext-whowrotethat-state-*` are InfoWidget states in general, and the error messages you're adding is literal error messages.

... or I'm just wasting your time with this super pedantic silliness. It can go either way :laughing:

Reply:
I love pedantry!

Done.